docs/ops/arithmetic/Sin_1.md

```diff
@@ -6,6 +6,18 @@
 
 **Short description**: *Sin* performs element-wise sine operation with given tensor.
 
+**Detailed description**: *sin* does the following with the input tensor *a*:
+\f[
+a_{i} = sin(a_{i})
+\f]
+
+The sine of an acute angle theta is defined in the context of a right triangle: for the specified angle, it is the ratio of the length of the side 
+that is opposite that angle (the opposite), to the length of the longest side of the triangle (the hypotenuse).
+
```

Comment:
Did you invent this description on your own or copy-pasted ? :)

Reply:
both :), but it is the easiest definition of the sine that I can think of.

Reply:
It would make much more sense if there was an option to add a image of the triangle to the documentation

Reply:
So maybe just redirect to [Wikipedia](https://en.wikipedia.org/wiki/Sine) ?

Reply:
if that is ok then sure